latex_table.py

```diff
@@ -17,7 +17,7 @@
     "HippyVM": "HippyVM",
     "PyHyp-mono": "PyHyp$_m$",
     "PyHyp-comp": "PyHyp$_c$",
-    "PyHyp-comp-rev": "PyHyp$_{c'}$",
+    "PyHyp-comp-rev": "PyHyp$_{c2}$",
```

Comment:
I don't mind too much. Inverse has the same implied meaning as reverse. I find `c'` concise. You choose.